en/SettingsView/General.md

```diff
@@ -27,51 +27,71 @@ This section defines a number of miscellaneous settings, related to (non exhaust
-  The process takes measurements over time, leading to increasing position accuracy.
-  Both of the setting conditions must met for the Survey-in process to complete.
-  For more information see [RTK GPS](https://docs.px4.io/en/advanced_features/rtk-gps.html) (PX4 docs) and [GPS- How it works](http://ardupilot.org/copter/docs/common-gps-how-it-works.html#rtk-corrections) (ArduPilot docs).
+The settings are:
+- **Disable all data persistence**: Check to prevent any data being saved or cached: logs, map tiles etc. 
+  This setting disables the [telemetry logs section](#telemetry_logs).
 
-<span></span>
-> **Tip** In order to save and reuse a base position (because Survey-In is time consuming!) perform Survey-In once, select *Use Specified Base Position* and press **Save Current Base Position** to copy in the values for the last survey.
-  The values will then persist across QGC reboots until they are changed.
+
+## Telemetry Logs from Vehicle {#telemetry_logs}
+
+![Telemetry Logs from Vehicle Settings](../../assets/settings/general/telemetry_logs.jpg)
 
 The settings are:
-- Perform Survey-In
-  - **Survey-in accuracy (U-blox only):** The minimum position accuracy for the RTK Survey-In process to complete.
-  - **Minimum observation duration:** The minimum time that will be taken for the RTK Survey-in process.
-- Use Specified Base Position
-  - **Base Position Latitude:** Latitude of fixed RTK base station.
-  - **Base Position Longitude:** Longitude of fixed RTK base station.
-  - **Base Position Alt (WGS94):** Altitude of fixed RTK base station.
-  - **Base Position Accuracy:** Accuracy of base station position information.
-  - **Save Current Base Position** (button): Press to copy settings from the last Survey-In operation to the *Use Specified Base Position* fields above.
+- <span id="autosave_log"></span>**Save log after each flight**: Telemetry logs (`.tlog`) automatically saved to the *Application Load/Save Path* ([above](#load_save_path)) after flight. 
+- **Save logs even if vehicle was not armed**: Logs when a vehicle connects to *QGroundControl*. 
+  Stops logging when the last vehicle disconnects.
+- **Save CSV log of telemetry data**: Log subset of telemetry data to a CSV file.
+
+
+## Fly View {#fly_view}
+
+![Fly View Settings](../../assets/settings/general/fly_view.jpg)
+
+The settings are:
+- **Use Preflight Checklist**: Enable pre-flight checklist in Fly toolbar.
+- **Enforce Preflight Checklist**: Checklist completion is a pre-condition for arming.
+- **Keep Map Centered on Vehicle**: Forces map to center on the currently selected vehicle.
+- **Show Telemetry Log Replay Status Bar**: Display status bar for [Replaying Flight Data](../FlyView/replay_flight_data.md).
+- **Virtual Joystick**: Enable [virtual joysticks](../SettingsView/VirtualJoystick.md) (PX4 only)
+- **Use Vertical Instrument Panel**: Align instrument panel vertically rather than horizontally (default).
+- **Show additional heading indicators on Compass**: TBD
```

Comment:
@DonLakeFlyer I tried setting additional heading indicators but can't tell what they do. Can you outline their purpose. Ideally with screenshot.

Reply:
![Screen Shot 2020-02-12 at 9 50 53 AM](https://user-images.githubusercontent.com/5876851/74362235-32fe5180-4d7d-11ea-81ed-5ef65711088c.png)

* Blue Array - Course Over Ground
* White House - Direction back to home
* Green Line - Direction to next waypoint

If you run Sitl, takeoff and do a go to location you can get these to move around